src/app/webui/components/Playground/PlaygroundView.tsx

```
@@ -0,0 +1,588 @@
+    setCurrentError(null);
+    setToolResult(null);
+
+    if (!validateInputs()) {
+      handleError("Please correct the input errors.", 'input');
+      return;
+    }
+
+    setIsLoading(true);
+    try {
+      const processedInputs: Record<string, any> = {};
+      if (selectedTool.inputSchema && selectedTool.inputSchema.properties) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)